pinot-connectors/pinot-flink-connector/src/test/resources/fixtures/pinotTableSchema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "schemaName": "demand",
+  "dimensionFieldSpecs": [
+    {
+      "name": "demand_uuid",
+      "singleValueField": true,
+      "dataType": "STRING",
+      "virtualColumnProvider": null,
+      "defaultNullValue": "null",
+      "transformFunction": null,
+      "maxLength" : 4096
+    },
+    {
+      "name": "geofence",
+      "singleValueField": true,
+      "dataType": "STRING",
+      "virtualColumnProvider": null,
+      "defaultNullValue": "null",
+      "transformFunction": null
+    }
+  ],
+  "metricFieldSpecs": [
+    {
+      "dataType": "DOUBLE",
+      "singleValueField": true,
+      "fieldSize": 8,
+      "derivedMetricType": null,
+      "name": "surge_multiplier",
+      "virtualColumnProvider": null,
+      "defaultNullValue": 0.0,
+      "transformFunction": null
+    }
+  ],
+  "dateTimeFieldSpecs": [
```

Comment:
how is this not failing with 2 `dateTimeFieldSpecs` !

Reply:
oh, it's just a dummy file to test the RPC calls